imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -71,3 +73,15 @@ func TestGuestSecureBoot(t *testing.T) {
 		t.Fatal("secure boot is not enabled as expected")
 	}
 }
+
+func TestBootTime(t *testing.T) {
+	metadata, err := utils.GetMetadataAttribute("start-time")
+	if err != nil {
+		t.Fatalf("couldn't get start time from metadata")
+	}
+	startTime, err := strconv.Atoi(metadata)
+	if err != nil {
+		t.Fatalf("failed to convet start time %s", metadata)
+	}
+	t.Logf("image boot time is %d", time.Now().Second()-startTime)
```

Comment:
we don't care how long it took?

Reply:
That's an open question. I didn't find a SLO for instance boot up.

Reply:
So i make this is just for log purpose

Reply:
Just a simple test point
```
Jul 27 19:58:49 vm2-image-boot-vd5tv GCEMetadataScripts[480]: 2021/07/27 19:58:49 GCEMetadataScripts: startup-script-url: === RUN   TestBootTime
Jul 27 19:58:49 vm2-image-boot-vd5tv GCEMetadataScripts[480]: 2021/07/27 19:58:49 GCEMetadataScripts: startup-script-url:     image_boot_test.go:86: image boot time is 26
Jul 27 19:58:49 vm2-image-boot-vd5tv GCEMetadataScripts[480]: 2021/07/27 19:58:49 GCEMetadataScripts: startup-script-url: --- PASS: TestBootTime (0.00s)
```
I make the threshold is 120 seconds.